packages/zk-transfer-web/src/App.tsx

```diff
@@ -13,7 +14,15 @@ type AppProp = {
 };
 
 function App({ demo } : AppProp) {
-  // await api.init(demo);
+  const [chainId, setDemoChain] = useState(demo.chainId);
+
+  useEffect(() => {
+    if (_.isUndefined(chainId))
```

Comment:
if this is the only use of `lodash`, let's avoid adding it as another dependency.
can we just do `if (chainId) {` ?